src/main/java/domain/Snapshot.java

```diff
@@ -10,6 +10,11 @@ public Snapshot(Object id, Object object) {
         this.object = object;
     }
 
+    public Snapshot(Snapshot snapshot) {
+        this.id = snapshot.getId();
+        this.object = snapshot.getObject();
```

Comment:
```suggestion
        this.id = snapshot.id
        this.object = snapshot.object
```

자기자신을 넘겨주면 변수 접근 가능하답니다 :)
이런 부분을 잘 활용하면, `getter`도 많이 줄일수 있어요 ㅎㅎ

Reply:
헉!!!! 처음 안 사실이에요.,......!!